Tools/CISupport/runUnittests.py

```diff
@@ -21,14 +18,44 @@ def run_unittests(test_discovery_path):
     results = unittest.TextTestRunner(buffer=True).run(test_suite)
     if results.failures or results.errors:
         raise RuntimeError('Test failures or errors were generated during this test run.')
+    return 0
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Run Python unit tests for our various build masters',
+    )
+    parser.add_argument(
+        'path', nargs=1,
+        help='Relative path of directory to run Python unit tests in',
+    )
+    parser.add_argument(
+        '--no-autoinstall',
```

Comment:
Let's keep the default behavior to not autoinstall for now (with the parameter --autoinstall passed for auto installation). If it is working fine without any issues in production, we can then enable autoinstall by default.